spyder/plugins/help/plugin.py

```diff
@@ -125,8 +125,7 @@ def on_shortcuts_available(self):
         shortcuts = self.get_plugin(Plugins.Shortcuts)
 
         # See: spyder-ide/spyder#6992
-        self._show_intro_message = lambda: self.show_intro_message()
-        shortcuts.sig_shortcuts_updated.connect(self._show_intro_message)
+        shortcuts.sig_shortcuts_updated.connect(self.show_intro_message)

```

Comment:
While checking this, I noticed that seems like spyder-ide/spyder#6992 is happening again due to the referencing of the wrong shortcut through `console/` instead of `ipython_console/` at https://github.com/spyder-ide/spyder/blob/a0d82eb7b62a86fe61c642cccac2f5db79706742/spyder/plugins/help/widgets.py#L788

Note totally related with the changes here but maybe that could be fixed here or should a new issue be created?

Reply:
Good catch @dalthviz! I'll fix it as part of this PR.